MLP_Model/MLP_utils/config.toml

```diff
@@ -22,4 +22,4 @@ LEARNING_RATE_MAX = 0.1
 OPTIMIZER_LIST = [ "Adam", "RMSprop", "SGD" ]
 METRIC = "loss"
 DIRECTION = "minimize"
-MODEL_NAME = "LPS_10_vs_DMSO"
+MODEL_NAME = "LPS_10_vs_LPS_100"
```

Comment:
just confirming. Whenever you are training a new model, you have to change this model name? And the `config.toml` file name never changes?

I wonder if the file name should also change, in case the parameters change per model. Oh, but maybe these parameters won't change? (It would still likely be good to keep a record of the models that you've tested)

One final point - is the model name arbitrary? In other words, it doesn't specify which treatments to include, it is simply an identifier?

Reply:
This is an issue. I am making this an issue now. I will address in a new PR. I will need to make my code more dynamic to parse the config to decide which treatments to test and also make different configs for each model. As of now the treatments selected are hardcoded into the notebook.